README.md

```diff
@@ -1,126 +1,249 @@
-    <html>
-        <p>The goal you entered was:</p>
-    </html>
-    <answer name="goal" read_only="true" />
+### Mentoring modes
 
-    <html>
-        <p>Ask feedback from friends about this goal - what did they think?</p>
-    </html>
-    <answer name="goal_feedback" />
-</mentoring>
-```
+There are 2 mentoring modes available:
 
-### Self-assessment quizzes
-
-```xml
-<mentoring url_name="quizz_1" enforce_dependency="false">
-    <quizz name="quizz_1_1" type="choices">
-        <question>Do you like this quizz?</question>
-        <choice value="yes">Yes</choice>
-        <choice value="maybenot">Maybe not</choice>
-        <choice value="understand">I don't understand</choice>
-
-        <tip display="yes">Great!</tip>
-        <tip reject="maybenot">Ah, damn.</tip>
-        <tip reject="understand"><html><div id="test-custom-html">Really?</div></html></tip>
-    </quizz>
-
-    <quizz name="quizz_1_2" type="rating" low="Not good at all" high="Extremely good">
-        <question>How much do you rate this quizz?</question>
-        <choice value="notwant">I don't want to rate it</choice>
-
-        <tip display="4,5">I love good grades.</tip>
-        <tip reject="1,2,3">Will do better next time...</tip>
-        <tip reject="notwant">Your loss!</tip>
-    </quizz>
-
-    <message type="completed">
-        All is good now...
-        <html><p>Congratulations!</p></html>
-    </message>
-</mentoring>
-```
+* *standard*: Traditional mentoring. All questions are displayed on the
+  page and submitted at the same time. The students get some tips and
+  feedback about their answers. This is the default mode.
+
+* *assessment*: Questions are displayed and submitted one by one. The
+  students don't get tips or feedback, but only know if their answer was
+  correct. Assessment mode comes with a default `max_attempts` of `2`.
+
+Below are some LMS screenshots of a mentoring block in assessment mode.
+
+Question before submitting an answer:
+
+![Assessment Step 1](doc/img/assessment-1.png)
+
+Question after submitting the correct answer:
+
+![Assessment Step 2](doc/img/assessment-2.png)
+
+Question after submitting a wrong answer:
+
+![Assessment Step 3](doc/img/assessment-3.png)
+
+Score review and the "Try Again" button:
+
+![Assessment Step 4](doc/img/assessment-4.png)
+
+### Free-form questions
+
+Free-form questions are represented by a "Long Answer" component. 
+
+Example screenshot before answering the question:
+
+![Answer Initial](doc/img/answer-1.png)
+
+Screenshot after answering the question:
+
+![Answer Complete](doc/img/answer-2.png)
+
+You can add "Long Answer Recap" components to mentoring blocks later on
+in the course to provide a read-only view of any answer that the student
+entered earlier.
+
+The read-only answer is rendered as a quote in the LMS:
+
+![Answer Read-Only](doc/img/answer-3.png)
+
+### Multiple Choice Questions
+
+Multiple choice questions can be added to a mentoring component and
```

Comment:
Capitalize 'choice' and 'questions' here, and put '(MCQ)' after the title in this header to make it especially clear what it stands for.